core/src/main/resources/lib/form/descriptorRadioList.jelly

```diff
@@ -48,14 +48,14 @@ THE SOFTWARE.
   <j:set var="targetType" value="${attrs.targetType?:it.class}"/>
   <f:section title="${attrs.title}">
     <d:invokeBody />
-	  <j:forEach var="d" items="${descriptors}" varStatus="loop">
+    <j:forEach var="d" items="${descriptors}" varStatus="loop">
       <f:radioBlock name="${varName}" help="${d.helpFile}" value="${loop.index}"
-        title="${d.displayName}" checked="${instance.descriptor==d}">
+                    title="${d.displayName}" checked="${instance.descriptor==d}">
```

Comment:
this looks like an indentation error

Reply:
I'm indenting to match the first attribute of the tag on the previous line. -- I'm not sure there's an actual style.